backup-stores/s3/src/test/java/io/camunda/zeebe/backup/s3/S3BackupStoreTests.java

```diff
@@ -235,4 +238,22 @@ default void deletingInProgressBackupFails(final Backup backup) {
         .withThrowableOfType(Throwable.class)
         .withRootCauseInstanceOf(BackupInInvalidStateException.class);
   }
+
+  @ParameterizedTest
+  @ArgumentsSource(TestBackupProvider.class)
+  default void shouldNotOverwriteCompletedBackup(final Backup backup) {
```

Comment:
:wrench: This should be part of the testkit, IMO. All store implementations should satisfy this.

Reply:
Moved to testkit.